api/src/pcapi/routes/backoffice/templates/admin/category_subcategory.html

```diff
@@ -0,0 +1,62 @@
+{% from "components/forms.html" import build_form_fields_group with context %}
+{% extends "layouts/connected.html" %}
+{% block page %}
+  <div class="pt-3 px-5 pc-feature-flip">
+    <h2 class="fw-light">Liste des sous-catégories existantes</h2>
```

Comment:
```suggestion
    <h2 class="fw-light">Liste des sous-catégories</h2>
```
Sauf si demain, tu écris la liste des sous-catégories inexistantes

Reply:
`    <h2 class="fw-light">Liste des sous-catégories</h2>`
FAIT